src/utils/global/ambassadorCard/AmbassadorCard.tsx

```diff
@@ -19,6 +22,24 @@ export default function AmbassadorCard(props: AmbassadorCardProps) {
   const images = getAmbassadorImages(ambassadorKey)
   const mod = window?.Twitch?.ext?.viewer?.role === 'broadcaster' || window?.Twitch?.ext?.viewer?.role === 'moderator'
 
+
+  const iucnText = "An objective assessment system for classifying the status of plants, animals, and other organisms threatened with extinction."
+  const iucnInfoIcon = (
+    <Tooltip
+      text={iucnText}
+      title="IUCN"
+      leftOffset="60%"
```

Comment:
this is still necessary because I tried to `position: relative` the wrapper and children and `position: absolute` the tooltip, which worked for the buttons

but, in the case of the ambassador card, this caused the card to contain the tooltip resulting in an overflow-x instead of using the `position: absolute` style which i thought should ignore its container and allow position based on its parent